returnn/tf/updater.py

```diff
@@ -926,9 +942,11 @@ def get_apply_grads_op(self, loss, var_list):
     return tf.group(*all_apply_grads)
 
 
-class KerasOptimizer(Optimizer):
+class KerasOptimizerWrapper(Optimizer):
   """
   Wraps a TF optimizer into a standard TF optimizer.
+
+  For internal use only.
```

Comment:
This comment is kind of obsolete when you directly use the class name `_KerasOptimizerWrapper` (where the `_` prefix exactly implies this).